### PR TITLE
Version v7.0.1-beta

### PR DESCRIPTION
**Pull request by jhendrixMSFT**

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
@mcardosos @marstr Here's the updated SDK with the flattening fix.

File: arm/cognitiveservices/accounts.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func (client AccountsClient) Create(resourceGroupName string, accountName string
 			Constraints: []validation.Constraint{{Target: "parameters.Sku", Name: validation.Null, Rule: true,
 				Chain: []validation.Constraint{{Target: "parameters.Sku.Tier", Name: validation.ReadOnly, Rule: true, Chain: nil}}},
 				{Target: "parameters.Location", Name: validation.Null, Rule: true, Chain: nil},
-				{Target: "parameters.Properties", Name: validation.Null, Rule: true, Chain: nil}}}}); err != nil {
+				{Target: "parameters.map[string]interface{}", Name: validation.Null, Rule: true, Chain: nil}}}}); err != nil {
```
Comment (mcardosos) [inline arm/cognitiveservices/accounts.go]: I wonder if this actually works? Maybe the problem comes from a bad defined swagger?

**Comment by tombuildsstuff** [comment]
@jhendrixMSFT out of interest, given the size of these PR's, what kind of tests are you running to validate this?

File: arm/cognitiveservices/accounts.go (hunk 1 above)
Comment (mcardosos) [inline arm/cognitiveservices/accounts.go]: Found the [guilty line](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json#L582)

Comment (jhendrixMSFT) [inline arm/cognitiveservices/accounts.go]: @mcardosos and I debugged this, it was a bug in the support for flattening.

**Comment by jhendrixMSFT** [comment]
@tombuildsstuff Unfortunately our coverage is a bit ad-hoc right now which is how this regression slipped in.  :(  We have a work item to improve our coverage and get it into our CI for the next release.

**Pull request merged**